cpp/oneapi/dal/backend/primitives/ndarray.hpp

```diff
@@ -277,6 +351,82 @@ class ndview : public ndarray_base<axis_count, order> {
     bool data_is_mutable_;
 };
 
+#ifdef ONEDAL_DATA_PARALLEL
+template <typename T1, ndorder ord1, typename T2, ndorder ord2>
+sycl::event copy_by_value(sycl::queue& q,
```

Comment:
copy_by_value -> copy, you introducing separate method, which makes it clear that it isn't just ref copy.

Reply:
fixed